internal/daemon/daemon.go

```diff
@@ -701,7 +701,17 @@ func New(opts *Options) (*Daemon, error) {
 		normalSocketPath:    opts.SocketPath,
 		untrustedSocketPath: opts.SocketPath + ".untrusted",
 	}
-	ovld, err := overlord.New(opts.Dir, d, opts.ServiceOutput)
+
+	exitPebble := make(chan struct{})
```

Comment:
This channel is wired up so that the servstate manager can close the channel to gracefully stop the Pebble server when "on-success: exit-pebble" or similar is specified in the service config.

Reply:
Thanks. It does need to be wired up somehow: at the end of the day `daemon.Stop()` needs to be called, and servstate is a dependency of overlord (which is a dependency of daemon), not the other way around. But I've simplified this a bit by wiring up `daemon.Stop` a bit more directly.